### PR TITLE
Fix multi level area template

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: BackofficeBundle/EventSubscriber/NodeTemplateSelectionSubscriber.php [hunk 1]
```diff
@@ -122,4 +116,19 @@ protected function getTemplateChoices()
 
         return $templatesChoices;
     }
+
+    /**
+     * @param AreaContainerInterface $areaContainer
+     * @param Collection             $areas
+     */
+    protected function hydrateAreaFromTemplate(AreaContainerInterface $areaContainer, $areas)
```
Comment (nicolasThal) [inline BackofficeBundle/EventSubscriber/NodeTemplateSelectionSubscriber.php]: I would rather call the second parameter `$sourceAreas`

**Comment by itkg-alavieille** [comment]
@nicolasThal

**Comment by itkg-ngilain** [comment]
:+1:

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**